src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Sse41.cs

```diff
@@ -182,12 +182,12 @@ public static class Sse41
         /// int _mm_extract_epi8 (__m128i a, const int imm8)
         ///   PEXTRB reg/m8, xmm, imm8
         /// </summary>
-        public static sbyte Extract(Vector128<sbyte> value, byte index) => Extract(value, index);
+        public static int Extract(Vector128<sbyte> value, byte index) => Extract(value, index);
```

Comment:
Also fixes SSE2/SSE4.1/AVX `Extract` return type to reflect the underlying instruction behavior https://github.com/dotnet/coreclr/issues/17957

cc @CarolEidt @tannergooding